v3/util/iam-proxy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source /etc/environment
+
+if [ "${NODE_ROLE}" != "worker" ]; then
+    exit 0
+fi
+
+docker pull "behance/iam-docker:latest"
```

Comment:
@bryanlatten agreed, I will make a card to update them all throughout the scripts together